gcloud/monitoring/timeseries.py

```diff
@@ -155,3 +199,27 @@ def _from_dict(cls, info):
             value = int(value)  # Convert from string.
 
         return cls(end_time, start_time, value)
+
+    def _to_dict(self):
+        """Build a dictionary ready to be serialized to the JSON wire format.
+
+        This method serializes a point in JSON format to be written
+        to the API.
+
+        :rtype: dict
+        :returns: The dictionary representation of the point object.
+        """
+        info = {
+            'interval': {
+                'endTime': _datetime_to_rfc3339(self.end_time)
+            },
+            'value': {
+                _get_typed_value_from_value(self.value): str(self.value)
```

Comment:
Why are you converting the value to a string? According to the following link, only strings and integers are expected to be strings.

https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue